src/Mvc/src/Microsoft.AspNetCore.Mvc.NewtonsoftJson/NewtonsoftJsonControllerExtensions.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Newtonsoft.Json;
+
+namespace Microsoft.AspNetCore.Mvc
+{
+    public static class NewtonsoftJsonControllerExtensions
+    {
+        /// <summary>
+        /// Creates a <see cref="JsonResult"/> object that serializes the specified <paramref name="data"/> object
+        /// to JSON.
+        /// </summary>
+        /// <param name="controller">The controller.</param>
+        /// <param name="data">The object to serialize.</param>
+        /// <returns>The created <see cref="JsonResult"/> that serializes the specified <paramref name="data"/>
+        /// to JSON format for the response.</returns>
+        public static JsonResult Json(this Controller controller, object data)
```

Comment:
This is kinda bogus. To use it you have to do `return this.Json(...)` in your controller, but it works

Reply:
Yeah, I think this is an unacceptable break. We need to keep these methods in place as regular methods, and change the parameter type to `object`.

Reply:
`public static ActionResult Json(this Controller controller, object data, object jsonSerializerSettings)`?